lib/pipeline.js

```diff
@@ -86,20 +87,50 @@ class PipelineModel extends BaseModel {
         this.scmContext = config.scmContext;
     }
 
+    /**
+     * Get the screwdriver configuration for the pipeline at the given ref
+     * @method _getPipelineFactory
+     * @return {Object}         PipelineFactory instance
+     */
+    _getPipelineFactory() {
+        // Lazy load factory dependency to prevent circular dependency issues
+        // https://nodejs.org/api/modules.html#modules_cycles
+        /* eslint-disable global-require */
```

Comment:
Should either eslint-enable afterwards or use `// eslint-disable-next-line global-require`